style.css

```diff
@@ -1 +1,285 @@
-/*  Add your styling here */
+body {
+    border: 3px solid black;
+}
+
+.container {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: 0.3fr 0.3fr 0.5fr 0.5fr 0.5fr 0.5fr 0.5fr;
+    grid-gap: 2px;
+
+    grid-template-areas:
```

Comment:
Nice experimentation with named areas as opposed to using `grid-column-start` etc (which you do at the larger breakpoint). If you were building this "for real" then I'd suggest sticking to _either_ named areas or `grid-column-start` and related properties, so it's easy to reason about how the layout changes between the two breakpoints.